cabal-testsuite/PackageTests/Check/ConfiguredPackage/COptions/CxxOptionsExtraLibraries/cabal.out

```diff
@@ -1,4 +1,6 @@
 # cabal check
 Warning: The following warnings are likely to affect your build negatively:
 Warning: Instead of 'cxx-options: -lgame' use 'extra-libraries: game'
+Warning: These warnings may cause trouble when distributing the package:
+Warning: Please consider including a changelog file in the 'extra-doc-files' section of the .cabal file if it contains useful information for users of the package.
```

Comment:
I don't understand. The code (and some explanations - like the changelog entry itself), that the warning is shown if the changelog file exists. But in this and other test-cases it doesn't.

Why is the warning shown?

Reply:
See updated changelog.